File: server/static/kotlin/v2/web/src/main/java/xyz/hoper/util/PropertiesUtil.java
package xyz.hoper.util;

import java.io.BufferedInputStream;
import java.io.FileInputStream;
import java.io.IOException;
import java.io.InputStream;
import java.util.Properties;

/**
 * 资源文件读取工具
 */
public class PropertiesUtil {

    private final Properties pps;

    public PropertiesUtil(String filePath) throws IOException {
        pps = new Properties();
        InputStream in = new BufferedInputStream(new FileInputStream(filePath));
        pps.load(in);
    }

    public String getString(String key) {
        return pps.getProperty(key);
    }


}

File: server/static/kotlin/v2/web/src/main/kotlin/xyz/hoper/utils/common/PropertiesUtil.kt
package xyz.hoper.utils.common

import java.io.BufferedInputStream
import java.io.FileInputStream
import java.io.InputStream
import java.util.*

/**
 * 资源文件读取工具
 */
class PropertiesUtil(filePath: String?) {
    private val pps: Properties
    fun getString(key: String?): String {
        return pps.getProperty(key)
    }

    init {
        pps = Properties()
        val `in`: InputStream = BufferedInputStream(FileInputStream(filePath))
        pps.load(`in`)
    }
}